batch/src/main/java/org/frankframework/batch/AbstractRecordHandler.java

```diff
@@ -43,6 +43,7 @@
  * @author  John Dekker
  * @deprecated Warning: non-maintained functionality.
  */
+@Deprecated
```

Comment:
Dit hebben wij ooit expres niet gedaan omdat het dan uit de frankdoc verdwijnt en we nog geen alternatief hebben.